.github/workflows/test-integration-unreleased.yml

```diff
@@ -17,18 +17,23 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        java: [ 11 ]
+        entry:
+          - { opensearch_ref: '1.x', java: 11 }
```

Comment:
Do we want to test against `1.x` branch as well?

Reply:
We may be releasing security patches, e.g. 1.3.2, so yes.